theme/functions.php

```diff
@@ -28,10 +28,22 @@ function mytheme_setup_theme_supported_features() {
 
 	// Define a palette
 	add_theme_support( 'editor-color-palette',
-			'#334499',
-			'#d0a5db',
-			'#eee',
-			'#444'
+		array(
+			'name' => 'strong blue', // theme should define has-strong-blue-background-color and has-strong-blue-color
+			'color' => '#334499',
+		),
+		array(
+			'name' => 'light grayish magenta',
+			'color' => '#d0a5db',
+		),
+		array(
+			'name' => 'very light gray',
```

Comment:
“Very” seems an odd descriptor here, but okay if there is also a “light gray”. 🤷🏻‍♂️